src/config.js

```diff
@@ -142,7 +134,18 @@ exports.loadModule = function(moduleName, onLoad) {
             }
         });
     } else {
-        load(module || exports.$loaded[moduleName]);
+        // backwards compatibility for node and packaged version
+        try {
+            loadedModule = this.$require(moduleName);
+        } catch (e) {}
```

Comment:
is it good to catch all errors? would it not be better to check whether the method exists or something similar

Reply:
require itself may throw errors when moduleName is not loaded yet, this basically restores the old code, before my commit that introduced the regression. I could also move try catch into $require itself, but then i'd have to add that in two places.